lib/modules/manager/kubernetes/extract.ts

```diff
@@ -14,6 +15,20 @@ export function extractPackageFile(content: string): PackageFile | null {
     return null;
   }
 
+  const deps: PackageDependency[] = [
+    ...extractImages(content),
+    ...extractApis(content),
+  ];
+
+  if (!deps.length) {
+    return null;
+  }
+  return { deps };
+}
```

Comment:
```suggestion  
  return deps.length ? { deps } : null;
}
```

simplify and needs coverage 😉